azurerm/data_source_mssql_server_blob_auditing_policies.go

```diff
@@ -0,0 +1,100 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmMsSqlServerBlobAuditingPolicies() *schema.Resource {
```

Comment:
taking a look through we can remove this Data Source since this doesn't make sense for this Resource

Reply:
OK